rbkit-lib/model/cpustorage.cpp

```diff
@@ -0,0 +1,141 @@
+#include "cpustorage.h"
+#include <QVariant>
+#include <QDebug>
+#include "cpumapping.h"
+
+void CpuStorage::addNewNode(QMap<int, QVariant> data)
+{
+    //qDebug() << data;
+    CpuNodePtr newNode( new CpuNode(data[RBKit::CeMethodName].toString(),
+                data[RBKit::CeLabel].toString(),
+                data[RBKit::CeFile].toString(),
+                data[RBKit::CeThreadId].toString(),
+                data[RBKit::CeLine].toDouble(),
+                data[RBKit::CeSingletonMethod].toInt())
+            );
+
+    CpuStorage::updateNewNodeLocation(data[RBKit::CeMethodName].toString(), newNode);
```

Comment:
methodName, label etc. are private properties [here](https://github.com/code-mancers/rbkit-client/blob/cpuprof-conventions/rbkit-lib/model/cpunode.h#L13)
i would then have to create an accessor method for that
if it's better then i'll create one